server/protocols/amnezia.py

```
@@ -0,0 +1,59 @@
+            "pid": None,
+            "started_at": None,
+            "version": status.get("version", ""),
+        })
+        await add_log("INFO", self.PROTOCOL_NAME, "Amnezia stopped")
+        return True
+
+    async def is_running(self) -> bool:
+        status = await get_core_status(self.PROTOCOL_ID)
+        return status.get("status") == "running"
+
+    async def add_client(self, username: str, client_data: dict) -> dict:
```

Ruff (0.15.2)

[warning] Unused method argument: `client_data`

(ARG002)